cocos/platform/CCPlatformConfig.h

```diff
@@ -154,6 +154,7 @@ THE SOFTWARE.
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
 #ifndef __MINGW32__
 #pragma warning (disable:4127) 
+#pragma warning (disable:4018)
```

Comment:
just on windows.and I see it from cocos2dx-0.x to now.never have seen anybody fixed it.Are you sure it will be fixed.And I think if you want to fix it ,then you can open it that day.. @andyque